tasks/licenses.py

```diff
@@ -72,11 +73,39 @@ def get_licenses_list(ctx):
     try:
         licenses = wwhrd_licenses(ctx)
         licenses = find_copyright(ctx, licenses)
-        return licenses_csv(licenses)
+        licenses = licenses_csv(licenses)
+        _verify_unknown_licenses(licenses, licenses_filename)
+
+        return licenses
     finally:
         shutil.rmtree("vendor/")
 
 
+def _verify_unknown_licenses(licenses, licenses_filename):
+    """
+    Check that all deps have a non-"UNKNOWN" copyright and license
+    """
+    unknown_licenses = False
+    for line in licenses:
+        parts = [part.strip().casefold() for part in line.split(',')]
+        if 'unknown' in parts:
+            unknown_licenses = True
+            print(f"! {line}")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Was the `print` left on purpose ? If it wasn't:
```suggestion
        unknown_licenses = unknown_licenses and 'unknown' in parts
```

Reply:
This print was already there before, it is useful to print each unknown license (if multiple). Imo it is better to leave it here